cli/azd/main.go

```diff
@@ -72,16 +73,44 @@ func main() {
 			// case
 			log.Printf("eliding update message for dev build")
 		} else if latestVersion.GT(internal.VersionInfo().Version) {
-			var upgradeUrl string
+			var upgradeText string
+
+			installedBy := installer.InstalledBy()
 			if runtime.GOOS == "windows" {
-				upgradeUrl = "https://aka.ms/azd/upgrade/windows"
+				switch installedBy {
+				case installer.InstallTypePs:
+					upgradeText = "run:\npowershell -ex AllSigned -c \"Invoke-RestMethod 'https://aka.ms/install-azd.ps1' | Invoke-Expression\"\n\nIf the install script was run with custom parameters, ensure that the same parameters are used for the upgrade. For advanced install instructions, see: https://aka.ms/azd/upgrade/windows"
```

Comment:
Can we add some text around the URL like,  "Visit <url> for more info" for all OSs?

@Austinauth Do URLs need to be stylized as URLs in warning/error messages?

Reply:
`For advanced install instructions, see: ...`?